libraries/Microsoft.Bot.Builder.Core.Extensions/TranscriptLoggers.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Core.Extensions
+{
+    /// <summary>
+    /// TraceTranscriptLogger, writes activites to Trace output
+    /// </summary>
+    public class TraceTranscriptLogger : ITranscriptLogger
+    {
+        private static JsonSerializerSettings serializationSettings = new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore, Formatting = Formatting.Indented };
```

Comment:
same comments here - consider using the same JSON Serialization settings as in other spots in our code.